docs/MD.md

```diff
@@ -1,29 +1,129 @@
 # Maskwacîs Dictionary
 
-## General Notes
-
-The TSV version of the Maskwacîs dictionary in the ALTLab repo also has semantic classifications, using the Rapid Words ontology.
+Notes on the Maskwacîs dictionary database.
 
 ## SRO
 
-* Inconsistent marking of vowel length.
+A transcription of the headword in Standard Roman Orthography (SRO).
+
+* Occasionally includes punctuation, **ex:** _ah?_ 'What? What did you say?'.
+* Vowel length is indicated with double vowels rather than macrons or circumflexes, **ex:** `aa`.
+* Vowel length is not consistently indicated. Many long vowels are written as short vowels.
```

Comment:
Vowel length is not indicated at all in MD, as we originally received the spreadsheet.

Reply:
I found four instances of doubled vowels in the data:

* `ikatestaa	ᐃᑲᑌᐢᑕᐊ	vc	Put it aside somewhere else.`
* `sihtoskaawew	ᓯᐦᑐᐢᑲᐊᐁᐧᐤ	vp	He supports him.`
* `kiipahikan	ᑭᐃᐸᐦᐃᑲᐣ	n	A lid, cork or stopper.`
* `kiipiskah	ᑭᐃᐱᐢᑲᐦ	vc	Block the way.`
* `kiisapiskisam	ᑭᐃᓴᐱᐢᑭᓴᒼ	v	He heats a metal object.`

It seems safe to assume those are intended as long vowels? Although I notice that the Syllabic transcriptions of all of these items do _not_ have long vowels.

Reply:
Yes, one could presume that's the intention, though one could also consider them typos.

I was able to match all of this with CW content in itwêwina, and they're not always spelled with the long vowel, e.g.:

https://itwewina.altlab.app/search?q=ikatestaa
https://itwewina.altlab.app/search?q=kisapiskisam

To really determine the vowel length, we'd need to listen to the recording as well as match with the linguistic analysis.